mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -140,6 +147,92 @@ properties:
       - :PUBLIC_WEBSITE
     immutable: true
     required: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'documentProcessingConfig'
+    immutable: true
```

Comment:
`documentProcessingConfig` is not documented as immutable in [REST Reference](https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/projects.locations.collections.dataStores), but the field is immutable (evidence: PATCH request returns
Field \"updateMask\" contains an immutable path \"document_processing_config\".)